ome-xml/src/main/java/ome/xml/meta/MetadataConverter.java

```diff
@@ -106,6 +106,267 @@ public static void convertMetadata(MetadataRetrieve src, MetadataStore dest) {
     convertRootAttributes(src, dest);
   }
 
+  /**
+   * Copy all metadata from the specified channel in the source MetadataRetrieve
+   * to the specified channel in the destination MetadataStore.  The specified
+   * destination Image must exist (e.g. via calling
+   * <code>setImageID(..., destImage)</code>), but the destination Channel only
+   * needs to exist if the copyID flag is <code>false<code>
+   * LightSource references are assumed to be correct and are not checked for validity.
+   *
+   * @param src the source {@link MetadataRetrieve}
+   * @param srcImage the index of the Channel's parent Image in src
+   * @param srcChannel the index of the Channel in src
+   * @param dest the destination {@link MetadataStore}
+   * @param destImage the index of the Channel's parent Image in dest
+   * @param destChannel the index of the Channel in dest
+   * @param copyID false if the Channel ID should be omitted from the copy operation
+   */
+  public static void convertChannelMetadata(MetadataRetrieve src, int srcImage, int srcChannel,
+    MetadataStore dest, int destImage, int destChannel, boolean copyID)
```

Comment:
Most `convert` methods act on `(MetadataRetrieve, MetadaStore)` arguments with the exception of `convertLightSources(MetadataRetrieve, MetadataStore, int, List<String>)`. 

Two questions:
- do we want to unify the argument ordering?
- is there a concrete use case for have `srcImage` and `destImage` rather than `imageIndex` (https://github.com/openmicroscopy/bioformats/pull/3243/files has the same image index but different channel indices).

Reply:
Should the new argument ordering then be ```MetadataRetrieve src, MetadataRetrieve dest, int srcImage, int srcChannel, int destImage, int destChannel, boolean copyID)```?

I can imagine a case for needing both ```srcImage``` and ```destImage```.  For HCS datasets with one plane per TIFF/STK/etc., copying all of the data from Image 0 Channel 0 in the helper reader's  ```MetadataRetrieve``` to some other Image/Channel in the HCS reader's ```MetadataStore``` could be useful.  If a simpler signature with only ```imageIndex``` would help, though, it could easily be added and just delegate appropriately.